WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/PostsCardViewModel.swift

```diff
@@ -83,8 +94,28 @@ private extension PostsCardViewModel {
         return filterSettings.currentPostListFilter().sortDescriptors
     }
 
+    func sync() {
+        let filter = filterSettings.currentPostListFilter()
+
+        let options = PostServiceSyncOptions()
+        options.statuses = filter.statuses.strings
+        options.number = Constants.numberOfPostsToSync
+        options.purgesLocalSync = true
+
+        postService.syncPosts(
+            ofType: .post,
+            with: options,
+            for: blog,
+            success: { _ in
+
```

Comment:
[q] Are you planning on reloading the tableview in the success closure?

Reply:
It's not needed. `NSFetchedResultsController` will automatically update the `tableView` based on the data on the database.

Once the news posts are saved in the dabase, it will automatically update it. :)